cnxepub/formatters.py

```diff
@@ -571,12 +571,14 @@ def html_listify(tree, root_xl_element, extensions, list_type='ol'):
     for node in tree:
         li_elm = etree.SubElement(root_xl_element, 'li')
         if node['id'] == 'subcol':
-            span_elm = etree.SubElement(li_elm, 'span')
-            span_elm.text = node['title']
+            span_elm = lxml.html.fragment_fromstring(
+                node['title'], create_parent='span')
```

Comment:
Ok. yes I see that now. Hmm. We may need to change that if Helene et al object, but it's the minimal change for now, agreed.